src/components/CurrentPathwayProvider.tsx

```diff
@@ -68,8 +72,41 @@ export const CurrentPathwayProvider: FC<CurrentPathwayProviderProps> = memo(({ c
   );
 
   useEffect(() => {
-    if (pathway) updatePathway(pathway);
-  }, [pathway, updatePathway]);
+    if (pathway) {
+      updatePathway(pathway).then(() => cache.invalidateQueries('pathways'));
+    }
+  }, [cache, pathway]);
+
+  // Update criteriaSource if criteria change
+  useEffect(() => {
+    if (!pathway) return;
+
+    let updated = false;
```

Comment:
More a reminder to myself than anything else, but I assume this will use `updatePathwayCriteriaSources` once #60 is merged.

Reply:
Yes, I had to make at least some change since the PathwayProvider has been deleted. Instead of trying to copy over all of the changes as they occur I am planning on waiting until #60 is merged.

Reply:
After reviewing Dylan's comment on #60 this shouldn't be necessary in this component (although the change needs to be carried over into `StaticApp/CurrentPathwayProvider`